include/bluetooth/services/ots_client.h

```diff
@@ -1,28 +1,51 @@
 /* @file
- * @brief Object Transfer client header
+ * @brief Object Transfer Service client header
  *
- * For use with the Object Transfer Service client (OTC)
+ * For use with the Object Transfer Service Client
  *
- * Copyright (c) 2020 - 2021 Nordic Semiconductor ASA
+ * Copyright (c) 2020 - 2022 Nordic Semiconductor ASA
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#ifndef ZEPHYR_INCLUDE_BLUETOOTH_HOST_AUDIO_OTC_H_
-#define ZEPHYR_INCLUDE_BLUETOOTH_HOST_AUDIO_OTC_H_
-
-/* TODO: Temporarily here - clean up, and move alongside the Object Transfer Service */
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_OTS_CLIENT_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_SERVICES_OTS_CLIENT_H_
+
+/**
+ * @brief Object Transfer Service Client (OTC)
```

Comment:
Could we add OTS discovery logic to this module? What do you think about this idea?

Then we can get rid of the bt_otc_indicate_handler API and use some other mechanism to notify API users that the OTS  client is ready (ots_client_ready_cb?).

Reply:
Something should be done with the discovery, yes.  But I would like to think about that.
The OTS is used as a secondary service, which complicates a bit.  But @Emil-Gydesen-Bose has as solution for included services in e.g. VCS, I believe.

Reply:
Created https://github.com/zephyrproject-rtos/zephyr/issues/42640